lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -66,6 +66,17 @@
             - "Specify if latest template version should be used, when running a stateless VM."
             - "If this parameter is set to I(true) stateless VM is created."
         version_added: "2.3"
+    storage_domain:
+        description:
+            - "Name of the storage domain where all template disks should be copied."
+            - "This parameter is considered only when template is provided."
+        version_added: "2.4"
+    disk_format:
+        description:
+            - "This parameter is considered only when template and storage domain is provided."
```

Comment:
Please also use proper doc formatting like: "C(template) and C(storage_domain) parameters are provided"